clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1099,12 +1103,24 @@ class raw2trace_t {
 
 #ifdef BUILD_PT_POST_PROCESSOR
         std::istream *kthread_file;
-        std::vector<syscall_pt_entry_t> pre_read_pt_entries;
```

Comment:
What was this used for before such that is it not needed now?  May be worth a mention in the PR description.

Reply:
See other comment. Will add to PR description.